.kilocode/workflows/archon-tasks.md

```
@@ -0,0 +1,189 @@
+---
+title: "Archon Pipeline Orchestrator — Automação MCP"
+last_updated: 2025-09-17
+form: automation-prompt
+tags: [archon, mcp, devops, pipeline, automation, kanban, orchestration]
+related:
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)